WordPress/JetpackStatsWidgets/Tracks/Tracks+StatsWidgets.swift

```diff
@@ -82,29 +78,37 @@ extension Tracks {
     // MARK: - Private Enums
 
     fileprivate enum ExtensionEvents: String {
-        // User installs an instance of the today widget
-        case todayWidgetUpdated = "today_home_extension_widget_updated"
-        // User installs an instance of the all time widget
-        case allTimeWidgetUpdated = "alltime_home_extension_widget_updated"
-        // Users installs an instance of the this week widget
-        case thisWeekWidgetUpdated = "thisweek_home_extension_widget_updated"
-        // Users installs an instance of the lockscreen today views widget
-        case todayViewsLockScreenWidgetUpdated = "today_views_lockscreen_extension_widget_updated"
-
-        case noEvent
-
-        static func widgetUpdated(for key: String) -> ExtensionEvents {
-            switch key {
-            case AppConfiguration.Widget.Stats.todayProperties:
-                return .todayWidgetUpdated
-            case AppConfiguration.Widget.Stats.allTimeProperties:
-                return .allTimeWidgetUpdated
-            case AppConfiguration.Widget.Stats.thisWeekProperties:
-                return .thisWeekWidgetUpdated
-            case AppConfiguration.Widget.Stats.lockScreenTodayViewsProperties:
-                return .todayViewsLockScreenWidgetUpdated
-            default:
-                return .noEvent
```

Comment:
This is one of the reasons of this refactoring. I want switch to happen on a type, such as `Widget.Stats.Kind` so compiler would notify about expanding tracking when we create a new widget kind.